src/solver/utils/filename.h

```diff
@@ -1,10 +1,57 @@
 #pragma once
 
 #include <string>
-#include <yuni/core/string.h>
+#include <memory>
 
-std::string getFilenameWithExtension(const YString& prefix,
-                                     const YString& extension,
-                                     uint year,
-                                     uint week,
-                                     uint optNumber = 0);
+// ------------------------------------
+// Optimization period : base class
+// ------------------------------------
+class optPeriod
+{
```

Comment:
// This class intends to provide an interface for the naming of optimization-related files (MPS files, etc.)

Reply:
Done